src/components/shifts/ShiftForm.vue

```diff
@@ -409,6 +417,10 @@ export default {
         );
       }
 
+      if (this.worktimeTooLong) {
+        messages.push(this.$t("shifts.warnings.maxWorktimeExceeded"));
```

Comment:
Die Soll-Arbeitszeit je Tag sind 8 Stunden, die 10 Stunden sind ja eigentlich nur Ausnahme unter bestimmten Randbedingungen. Was hältst Du von dem Ansatz, eine Warnung bei `8 < worktime < 10` als (gelbe) Warnung anzuzeigen?

Das ist auch auf der Progress-Card so ähnlich.

Reply:
(Ich weiß, dass wir uns trotzdem auf die 10h max geeinigt haben - das stelle ich auch nicht in Frage; wäre eher eine Ergänzung.)